src/content/migrate/4.md

```diff
@@ -37,7 +46,7 @@ module.exports = {
 }
 ```
 
-Alternatively you can pass it via CLI: `--mode production`/`--mode development`
+T> Since the goals of the `development` and `production` versions are different. You could use `webpack-merge` to use the best of both modes. Read more [here](/guides/production)
```

Comment:
I have missed this, but google accessibility test recommends us to avoid words like `here`, `see here`, `follow link` etc. Please change this:
```suggestion
T> The purpose of `development` mode and `production` mode is different. You can use `webpack-merge` as shown in [production guide](/guides/production/#setup) to optimize configurations.
```